server/server.js

```diff
@@ -34,37 +35,28 @@ app.get("/api/user", cors(), async (req, res) => {
   }
 });
 
-app.get("/api/join", cors(), async (req, res) => {
+app.get("/api/favorite/:id", cors(), async (req, res) => {
+  // const userID = req.query.users_id;
+  // const tradeID = req.params.id;
+  const userID = 1;
   try {
-    const { rows: trades } = await db.query(
-      "SELECT trades.name, trades.content, trades.link, users.username, users.id AS users_id FROM trades LEFT JOIN users ON trades.users_id = users.id"
+    const { rows: trade_id } = await db.query(
+      "SELECT trade_id FROM users_trades WHERE users_id = $1",
+      [userID]
     );
-    res.send(trades);
+    res.send(trade_id);
   } catch (e) {
     return res.status(400).json({ e });
```

Comment:
nit: user a 500 code here to catch a generic error. maybe santize the error message a bit if possible